github/data_source_github_repository_file.go

```diff
@@ -9,12 +9,13 @@ import (
 	"strings"
 
 	"github.com/google/go-github/v57/github"
-	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
 )
 
 func dataSourceGithubRepositoryFile() *schema.Resource {
 	return &schema.Resource{
-		Read: dataSourceGithubRepositoryFileRead,
+		ReadContext: dataSourceGithubRepositoryFileRead,
```

Comment:
In the future, all data sources and resources should use `{verb}Context` instead of just `{verb}` function definitions.